### DISCUSSION TITLE
Help an idiot out

### DISCUSSION THREAD

**Discussion by goodellcurtisPFG** [teslamate-org/teslamate#3437] [General]
Installed and configured Docker Desktop on a NUC a few months ago. Everything was running fine until a few weeks ago and then the grafana reports were not being updated. Do I need to do something to the YML file to make this work again. I've read through these comments, seeing that support is lacking, but I haven't been able to find a message whether there is a short term fix.

**Comment by DrMichael** [comment]
Did you a `docker compose pull` and then `docker compose up -d`?

**Reply by goodellcurtisPFG** [reply]
That fixed me. Thank you so much
